src/psyclone/domain/lfric/algorithm/lfric_alg.py

```diff
@@ -260,8 +261,12 @@ def _create_function_spaces(self, prog, fspaces):
         # The order of the finite-element scheme.
         fe_config_mod = table.new_symbol(
             "finite_element_config_mod", symbol_type=ContainerSymbol)
-        order = table.new_symbol(
-            "element_order", tag="element_order",
+        order_h = table.new_symbol(
```

Comment:
Grepping for `element_order` in codebase returns one occurence in this module in line 240, a part of the test description: "The order of these spaces is set by the element_order variable which is provided by the LFRic finite_element_config_mod module."
I would suggest instead: "The order of these spaces is set by the element_order_<h,v> variables which are provided by the LFRic finite_element_config_mod module."